tests/sar_pipeline/isce3_rtc/test_full_docker_build_and_run.py

```diff
@@ -294,3 +485,127 @@ def test_docker_dual_pol_with_args():
+)
+TEST_2_RTC_S1_STATIC_LOCAL_COMPARISON_OUTPUTS_DIR = (
+    f"{LOCAL_COMPARISON_OUTPUTS_DIR}/TEST_2_RTC_S1_STATIC"
+)
+
+
+def test_compare_dual_pol_rtc_s1_static_product_outputs():
+    """
+    This function will compare the outputs created in the test_docker_single_pol_with_args
+    function with existing outputs in AWS. The output of this are data that describe
+    the differences in product files, metadata (.json and xml) and in the tifs themselves.
+    These outputs can be used to understand if the changes made are acceptable.
+    """
+
+    logging.info(
+        f"COMPARE 1: Comparing created Static Layers with Existing Accepted Product"
+    )
+    logging.info(
+        f"Creating comparison outputs directory : {TEST_2_RTC_S1_STATIC_LOCAL_COMPARISON_OUTPUTS_DIR}"
+    )
+    os.makedirs(TEST_2_RTC_S1_STATIC_LOCAL_COMPARISON_OUTPUTS_DIR, exist_ok=True)
+
+    runner = CliRunner()
+    args = ["--product", "RTC_S1_STATIC"]
+    args += ["--s3-product-folder-1", TEST_2_RTC_S1_STATIC_S3_PROJECT_FOLDER]
+    args += ["--s3-product-folder-2", TEST_2_COMPARE_RTC_S1_STATIC_S3_PROJECT_FOLDER]
+    args += ["--s3-bucket", TEST_S3_BUCKET]
+    args += ["--out-folder", TEST_2_RTC_S1_STATIC_LOCAL_COMPARISON_OUTPUTS_DIR]
+
+    result = runner.invoke(compare_products, args, catch_exceptions=False)
+    if result.exception:
+        logging.exception(
+            "An error occurred during CLI invocation", exc_info=result.exception
+        )
+        logging.error(result.output)
+
+    assert result.exit_code == 0
+
+    # ensure that the files and tif values have not changed. If so, these are breaking changes to the
+    # product, and the persistent comparison products must be updated - see description at top
+    file_differences = (
+        f"{TEST_2_RTC_S1_STATIC_LOCAL_COMPARISON_OUTPUTS_DIR}/file_differences.json"
+    )
+    tif_differences = (
+        f"{TEST_2_RTC_S1_STATIC_LOCAL_COMPARISON_OUTPUTS_DIR}/tif_differences.json"
+    )
+    assert not _files_have_changed(
+        file_differences
+    ), f"Error, there are breaking changes in the test files compared to the comparison product. \
+    Check {file_differences} and update comparison products if needed."
+    assert not _tifs_have_changed(
+        tif_differences
+    ), f"Error, the created tif values have changed compared to the comparison product. \
+    Check {tif_differences} and update comparison products if needed."
+
+
+# folders for products to compare
+TEST_2_RTC_S1_S3_PROJECT_FOLDER = (
+    f"{TEST_S3_PROJECT_FOLDER}/{TEST_2_S3_RTC_S1_PRODUCT_SUBPATH}"
+)
+TEST_2_COMPARE_RTC_S1_S3_PROJECT_FOLDER = (
+    f"{PERSISTENT_S3_PROJECT_FOLDER}/{TEST_2_S3_RTC_S1_PRODUCT_SUBPATH}"
+)
+TEST_2_RTC_S1_LOCAL_COMPARISON_OUTPUTS_DIR = (
+    f"{LOCAL_COMPARISON_OUTPUTS_DIR}/TEST_2_RTC_S1"
+)
+
+
+def test_compare_dual_pol_rtc_s1_product_outputs():
+    """
+    This function will compare the outputs created in the test_docker_single_pol_with_args
```

Comment:
Specify that this is for RTC layers and update the function name in the docstring to say "dual_pol"